changelog

```diff
@@ -1,3 +1,400 @@
+      * When taking off from a planet with an outfitter with more outfits in cargo than there is space for, store them instead of selling them. (@warp-core)
+      * Ship variant definitions can now remove all the bays from the base definition. (@daggs1)
+      * It is now possible to offer a mission in the shipyard or outfitter. (@rovermmicrover)
+      * A system definition can now include a custom value for the invisible fence radius. (@a-random-lemurian)
+      * TODO: find a concise way to describe the foreign penalty thing for governments. (@Hurleveur)
+      * Missions can now define a "to accept" condition set. (@rovermicrover)
+      * Governments can use named "stock" colors instead of always defining a color literal. (@warp-core)
+      * Added a "flotsam chance" attribute for outfits which determines the chance the outfit will be dropped as flotsam when the ship carrying it explodes. (@Azure3141)
+      * Governments can define custom values for various actions against foreign governments. (@warp-core)
+      * Added an "optical jamming" attribute that works similarly to radar jamming, but against optically guided missiles. (@Hurleveur)
+      * Added the "invisible" system property which makes systems inaccessible. (@yjhn, @Amazinite)
+    * Mechanic changes:
+      * Frugal ships now consider their health percentage instead of the sum of their shield and hull percentages. (@Hurleveur)
+      * Changed the scanning formula for detecting illegal cargo so more legal cargo can conceal illegal cargo in the hold, making it less likely to be detected. (@quyykk)
+      * Ships with the "mining" personality no longer continue mining when they should flee. (@Hurleveur)
+      * Scan speeds are now affected by more factors. (@Ferociousfeind)
+      * Ships with the "forbearing" personality will now also react to special damage types, instead of just based on health. (@Hurleveur)
+      * Both a dialog and conversation may now be displayed upon completing the objective for a mission NPC. (@Hurleveur)
+      * Improve wormhole definition syntax. (@quyykk, @tibetiroka)
+        * TODO: this
+      * Ships no longer always have a small amount of ramscoop when very close to the centre of a system. (@Zitchas)
+      * Restored the old overheating behaviour where a ship is completely unable to act, instead of just not having energy generation while overheated. (@warp-core)
+      * Initiating a hyperspace jump with no travel plan set does not set one. (@Zitchas)
+      * Hazard environmental effects may now be given non-integer magnitudes. (@RisingLeaf)
+  * User interface:
+    * Added shortcut keys for parking and unparking ships. (@Terin)
+    * Added a setting to always underline the shortcut hotkey on button labels. (@warp-core)
+    * Display the jump range separately to the view range on the map. (@RisingLeaf)
```

Comment:
```suggestion
    * Display the jump range different from the view range on the map. (@RisingLeaf)
```
It was already displayed separately, all i did was changing the visual

Reply:
```suggestion
    * Changed the line style for the jump range in the map to differentiate it from the view range. (@RisingLeaf)
```

Reply:
yea, that is better